lib/apiservers/engine/backends/container.go

```diff
@@ -1543,6 +1546,48 @@ func ContainerSignal(containerID string, sig uint64) error {
 	return nil
 }
 
+func getPortInformation(t *models.ContainerInfo, names []string) []types.Port {
+	var ports []types.Port
+	if len(names) == 0 {
+		log.Errorf("Couldn't find container while looking up port bindings; No container names provided")
+		return ports
+	}
+
+	var container *viccontainer.VicContainer
+	container = cache.ContainerCache().GetContainer(names[0])
+	if container == nil {
+		if match := regexp.MustCompile(`[\w_]+`).FindString(names[0]); match != "" {
```

Comment:
Then it'd have to be global or package scope. Seems like a bad plan.
Also: how would it catch a RE expression at "first run" instead of "runtime"? Do you mean an error compiling the regex? Or an error performing a match? Because if you mean the latter, that needs to happen at runtime.